statistics-production/ud.qmd

```diff
@@ -438,7 +438,7 @@ In line with the [GSS guidance on symbols](https://gss.civilservice.gov.uk/polic
 
 | Symbol      | Usage                                         | Example                                                   | Obsolete equivalents |
 |-------------|-----------------------------------------------|-----------------------------------------------------------| -------------------- |
-| z           | When an observation is **not applicable**     | No data for at gender level for boys at an all-girls school | |
+| z           | When an observation is **not applicable**     | No data for at sex level for boys at an all-girls school | |
```

Comment:
Looks like this line needs a bit of rephrasing for it to make sense. Didn't make much sense before the change to be fair!

Reply:
@rmbielby have just removed those few words entirely - does that make more sense?

Reply:
Yep, that's a lot clearer now